[atk] Build failure

Package: atk[core,introspection]:arm64-windows -> 2.38.0#7

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 28483f795 2023-09-13 (52 seconds ago)

**To Reproduce**

`vcpkg install --enforce-port-checks`
**Failure logs**

```
-- Using cached GNOME-atk-2.38.0.tar.gz.
-- Extracting source C:/v/downloads/GNOME-atk-2.38.0.tar.gz
-- Using source at C:/v/vcpkg4/buildtrees/atk/src/2.38.0-91fb0d495c.clean
-- Found external ninja('1.11.0').
-- Getting CMake variables for arm64-windows
-- Configuring arm64-windows-dbg
-- Using cached mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using msys root at C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f
-- Configuring arm64-windows-dbg done
-- Configuring arm64-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: C:/v/downloads/tools/python/python-3.11.5-x64/python.exe C:/v/vcpkg4/vcpkg_installed/x64-windows/tools/meson/meson.py --buildtype plain --backend ninja --wrap-mode nodownload --native C:/v/vcpkg4/scripts/buildsystems/meson/none.txt --libdir lib -Dintrospection=true --cross C:/v/vcpkg4/buildtrees/atk/meson-arm64-windows-rel.log -Ddebug=false --prefix C:/v/vcpkg4/packages/atk_arm64-windows -Dcmake_prefix_path=['C:/v/vcpkg4/vcpkg_installed/arm64-windows','C:/v/vcpkg4/vcpkg_installed/arm64-windows/debug','C:/v/vcpkg4/vcpkg_installed/arm64-windows/share'] C:/v/vcpkg4/buildtrees/atk/src/2.38.0-91fb0d495c.clean
    Working Directory: C:/v/vcpkg4/buildtrees/atk/arm64-windows-rel
    Error code: 1
    See logs for more information:
      C:\v\vcpkg4\buildtrees\atk\config-arm64-windows-rel-meson-log.txt.log
      C:\v\vcpkg4\buildtrees\atk\config-arm64-windows-rel-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_meson.cmake:431 (vcpkg_execute_required_process)
  ports/atk/portfile.cmake:28 (vcpkg_configure_meson)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\v\vcpkg4\buildtrees\atk\config-arm64-windows-rel-meson-log.txt.log</summary>

```
Build started at 2023-09-13T17:44:41.086936
Main binary: C:\v\downloads\tools\python\python-3.11.5-x64\python.exe
Build Options: -Dintrospection=true -Ddebug=false '-Dcmake_prefix_path=['"'"'C:/v/vcpkg4/vcpkg_installed/arm64-windows'"'"','"'"'C:/v/vcpkg4/vcpkg_installed/arm64-windows/debug'"'"','"'"'C:/v/vcpkg4/vcpkg_installed/arm64-windows/share'"'"']' -Dprefix=C:/v/vcpkg4/packages/atk_arm64-windows -Dlibdir=lib -Dbackend=ninja -Dbuildtype=plain -Dwrap_mode=nodownload '--cross-file C:/v/vcpkg4/buildtrees/atk/meson-arm64-windows-rel.log' '--native-file C:/v/vcpkg4/scripts/buildsystems/meson/none.txt'
Python system: Windows
The Meson build system
Version: 0.62.1
Source dir: C:\v\vcpkg4\buildtrees\atk\src\2.38.0-91fb0d495c.clean
Build dir: C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel
Build type: cross build
Project name: atk
Project version: 2.38.0
Sanity testing C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe
Is cross compiler: True.
Sanity check compiler command line: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe sanitycheckc.c /Fesanitycheckc_cross.exe -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include /nologo /showIncludes /utf-8 /c /link
Sanity check compile stdout:
sanitycheckc.c

-----
Sanity check compile stderr:

-----
C compiler for the host machine: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe (msvc 19.37.32822 "Microsoft (R) C/C++ Optimizing Compiler Version 19.37.32822 for ARM64")
C linker for the host machine: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/link.exe link 14.37.32822.0
Compiler for language c for the build machine not found.
Build machine cpu family: x86_64
Build machine cpu: x86_64
Host machine cpu family: aarch64
Host machine cpu: armv8
Target machine cpu family: aarch64
Target machine cpu: armv8
Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2u_1jsie
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2u_1jsie\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2u_1jsie\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- -FImsvc_recommended_pragmas.h 

Code:
 extern int i;
int i;

Compiler stdout:
 testfile.c
C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2u_1jsie\testfile.c: fatal error C1083: Cannot open include file: 'msvc_recommended_pragmas.h': No such file or directory

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Compiler for C supports arguments -FImsvc_recommended_pragmas.h: NO 
Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp9wl79awk
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp9wl79awk\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp9wl79awk\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- -utf-8 

Code:
 extern int i;
int i;

Compiler stdout:
 testfile.c

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Compiler for C supports arguments -utf-8: YES 
Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2wx7hr_x
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -INCREMENTAL:NO -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2wx7hr_x\testfile.c /FeC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2wx7hr_x\output.exe /nologo /showIncludes /utf-8 /MD -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /Od /Oi- kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /link -machine:ARM64 -DEBUG -OPT:REF -OPT:ICF /LIBPATH:C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib 

Code:
 
        #define bind_textdomain_codeset meson_disable_define_of_bind_textdomain_codeset
        
        #include <limits.h>
        #undef bind_textdomain_codeset
        
        #ifdef __cplusplus
        extern "C"
        #endif
        char bind_textdomain_codeset (void);
        
        #if defined __stub_bind_textdomain_codeset || defined __stub___bind_textdomain_codeset
        fail fail fail this function is not going to work
        #endif
        
        int main(void) {
          return bind_textdomain_codeset ();
        }
Compiler stdout:
 cl : Command line warning D9030 : '/showIncludes' is incompatible with multiprocessing; ignoring /MP switch
testfile.c
Note: including file: C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\limits.h
Note: including file:  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
testfile.obj : error LNK2019: unresolved external symbol bind_textdomain_codeset referenced in function main
C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp2wx7hr_x\output.exe : fatal error LNK1120: 1 unresolved externals

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Checking for function "bind_textdomain_codeset" : NO 
Pkg-config binary for 1 is not cached.
Pkg-config binary for 1 specified from cross file, native file, or env var as ['C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe']
'utf-8' codec can't decode byte 0x90 in position 2: invalid start byte
Unusable script 'C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe'
Found pkg-config: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe (1.8.0)
Determining dependency 'glib-2.0' with pkg-config executable 'C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe'
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --modversion glib-2.0` -> 0
2.76.3
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --cflags glib-2.0` -> 0
-IC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../include/glib-2.0 -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../include -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../lib/glib-2.0/include
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --libs glib-2.0` -> 0
-LC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../lib -lglib-2.0 -lintl
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --libs glib-2.0` -> 0
-LC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../lib -lglib-2.0 -lintl
Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpni5zvijt
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpni5zvijt\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpni5zvijt\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) {
            void * something;
            return 0;
        }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h
C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpni5zvijt\testfile.c(4): warning C4101: 'something': unreferenced local variable

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpmpinbyds
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpmpinbyds\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpmpinbyds\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) >= 0)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpj52g9bx_
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpj52g9bx_\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpj52g9bx_\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 0)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp51ox6wlw
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp51ox6wlw\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp51ox6wlw\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 1)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpnxq4mq3_
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpnxq4mq3_\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpnxq4mq3_\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 3)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpk0172p3x
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpk0172p3x\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpk0172p3x\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 7)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpegdp_e1r
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpegdp_e1r\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpegdp_e1r\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) > 15)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h
C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpegdp_e1r\testfile.c(3): error C2118: negative subscript

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpcfnbe7w7
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpcfnbe7w7\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpcfnbe7w7\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) <= 11)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp9ebpqezz
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp9ebpqezz\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmp9ebpqezz\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) <= 9)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Running compile:
Working directory:  C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpqtikz2sm
Command line:  C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/include C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpqtikz2sm\testfile.c /FoC:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\tmpqtikz2sm\output.obj /nologo /showIncludes /utf-8 /c -nologo -DWIN32 -D_WINDOWS -W3 -utf-8 -MP -MD -O2 -Oi -Gy -DNDEBUG -Z7 /nologo /showIncludes /utf-8 /c /Od /Oi- 

Code:
 #include <stdio.h>
        
        int main(void) { static int a[1-2*!(sizeof(void *) <= 8)]; a[0]=0; return 0; }
Compiler stdout:
 testfile.c
Note: including file: C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\stdio.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt.h
Note: including file:   C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vcruntime.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\sal.h
Note: including file:     C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\concurrencysal.h
Note: including file:    C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vadefs.h
Note: including file:  C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_wstdio.h
Note: including file:   C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_stdio_config.h

Compiler stderr:
 cl : Command line warning D9025 : overriding '/O2' with '/Od'
cl : Command line warning D9025 : overriding '/Oi' with '/Oi-'

Run-time dependency glib-2.0 found: YES 2.76.3
Pkg-config binary for 1 is cached.
Determining dependency 'gobject-2.0' with pkg-config executable 'C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe'
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --modversion gobject-2.0` -> 0
2.76.3
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --cflags gobject-2.0` -> 0
-IC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../include -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../include/glib-2.0 -IC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../lib/glib-2.0/include
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --libs gobject-2.0` -> 0
-LC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../lib -lgobject-2.0 -lglib-2.0 -lintl
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --libs gobject-2.0` -> 0
-LC:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig//../../lib -lgobject-2.0 -lglib-2.0 -lintl
Run-time dependency gobject-2.0 found: YES 2.76.3
Configuring config.h using configuration
Configuring atkversion.h using configuration
Pkg-config binary for 0 is not cached.
Pkg-config binary for 0 specified from cross file, native file, or env var as ['false']
Did not find pkg-config by name 'false'
Found Pkg-config: NO
Pkg-config binary for machine 0 not found. Giving up.
WARNING: Could not detect glib version, assuming 2.54. You may get build errors if your glib is older.
Program glib-genmarshal found: YES
Program glib-mkenums found: YES
Program glib-mkenums found: YES
Configuring atk.rc using configuration
'utf-8' codec can't decode byte 0x90 in position 2: invalid start byte
Unusable script 'C:/Program Files (x86)/Windows Kits/10/bin/10.0.22621.0/x64/rc.exe'
Windows resource compiler: Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384

Pkg-config binary for 1 is cached.
Determining dependency 'gobject-introspection-1.0' with pkg-config executable 'C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe'
env[PKG_CONFIG]: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe
env[PKG_CONFIG_PATH]: C:/v/vcpkg4/vcpkg_installed/arm64-windows/lib/pkgconfig/;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share/pkgconfig/
Called `C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe --modversion gobject-introspection-1.0` -> 1

CMake binary for 1 is not cached
CMake binary for 1 specified from cross file, native file, or env var as ['C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe']
'utf-8' codec can't decode byte 0x90 in position 2: invalid start byte
Unusable script 'C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe'
Found CMake: C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe (3.27.1)
Extracting basic cmake information
CMake Toolchain: Calling CMake once to generate the compiler state
Calling CMake (['C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe']) in C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\__CMake_compiler_info__ with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-G"
  - "Ninja"
  - "-DCMAKE_TOOLCHAIN_FILE=C:/v/vcpkg4/buildtrees/atk/arm64-windows-rel/meson-private/__CMake_compiler_info__/CMakeMesonTempToolchainFile.cmake"
  - "."
  - "-DCMAKE_PREFIX_PATH=C:/v/vcpkg4/vcpkg_installed/arm64-windows;C:/v/vcpkg4/vcpkg_installed/arm64-windows/debug;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share"
CMake trace warning: add_executable() non imported executables are not supported
CMake TRACE: C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\__CMake_compiler_info__\CMakeFiles\CMakeScratch\TryCompile-yqf6em\CMakeLists.txt:21 add_executable(['cmTC_e1be3', 'C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/share/cmake-3.27/Modules/CMakeCCompilerABI.c'])
CMake trace warning: target_link_options() TARGET cmTC_e1be3 not found
CMake TRACE: C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\__CMake_compiler_info__\CMakeFiles\CMakeScratch\TryCompile-yqf6em\CMakeLists.txt:24 target_link_libraries(['cmTC_e1be3', ''])
Try CMake generator: auto
Calling CMake (['C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe']) in C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-private\cmake_gobject-introspection-1.0 with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-DCMAKE_TOOLCHAIN_FILE=C:/v/vcpkg4/buildtrees/atk/arm64-windows-rel/meson-private/cmake_gobject-introspection-1.0/CMakeMesonToolchainFile.cmake"
  - "."
  - "-DCMAKE_PREFIX_PATH=C:/v/vcpkg4/vcpkg_installed/arm64-windows;C:/v/vcpkg4/vcpkg_installed/arm64-windows/debug;C:/v/vcpkg4/vcpkg_installed/arm64-windows/share"
  -- Module search paths:    ['C:/Program Files', 'C:/Program Files (x86)', 'C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386', 'C:/v/vcpkg4/vcpkg_installed/arm64-windows', 'C:/v/vcpkg4/vcpkg_installed/arm64-windows/debug', 'C:/v/vcpkg4/vcpkg_installed/arm64-windows/share']
  -- CMake root:             C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/share/cmake-3.27
  -- CMake architectures:    []
  -- CMake lib search paths: ['lib', 'lib32', 'lib64', 'libx32', 'share']
Preliminary CMake check failed. Aborting.
Run-time dependency gobject-introspection-1.0 found: NO (tried pkgconfig and cmake)

..\src\2.38.0-91fb0d495c.clean\atk\meson.build:139:2: ERROR: Dependency "gobject-introspection-1.0" not found, tried pkgconfig and cmake
```
</details>
<details><summary>C:\v\vcpkg4\buildtrees\atk\config-arm64-windows-rel-out.log</summary>

```
WARNING: Recommend using either -Dbuildtype or -Doptimization + -Ddebug. Using both is redundant since they override each other. See: https://mesonbuild.com/Builtin-options.html#build-type-options
The Meson build system
Version: 0.62.1
Source dir: C:\v\vcpkg4\buildtrees\atk\src\2.38.0-91fb0d495c.clean
Build dir: C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel
Build type: cross build
Project name: atk
Project version: 2.38.0
C compiler for the host machine: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/cl.exe (msvc 19.37.32822 "Microsoft (R) C/C++ Optimizing Compiler Version 19.37.32822 for ARM64")
C linker for the host machine: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.37.32822/bin/Hostx64/arm64/link.exe link 14.37.32822.0
Compiler for language c for the build machine not found.
Build machine cpu family: x86_64
Build machine cpu: x86_64
Host machine cpu family: aarch64
Host machine cpu: armv8
Target machine cpu family: aarch64
Target machine cpu: armv8
Compiler for C supports arguments -FImsvc_recommended_pragmas.h: NO 
Compiler for C supports arguments -utf-8: YES 
Checking for function "bind_textdomain_codeset" : NO 
Found pkg-config: C:/v/downloads/tools/msys2/6f3fa1a12ef85a6f/mingw32/bin/pkg-config.exe (1.8.0)
Run-time dependency glib-2.0 found: YES 2.76.3
Run-time dependency gobject-2.0 found: YES 2.76.3
Configuring config.h using configuration
Configuring atkversion.h using configuration
Did not find pkg-config by name 'false'
Found Pkg-config: NO
WARNING: Could not detect glib version, assuming 2.54. You may get build errors if your glib is older.
Program glib-genmarshal found: YES
Program glib-mkenums found: YES
Program glib-mkenums found: YES
Configuring atk.rc using configuration
Windows resource compiler: Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384

Found CMake: C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe (3.27.1)
Run-time dependency gobject-introspection-1.0 found: NO (tried pkgconfig and cmake)

..\src\2.38.0-91fb0d495c.clean\atk\meson.build:139:2: ERROR: Dependency "gobject-introspection-1.0" not found, tried pkgconfig and cmake

A full log can be found at C:\v\vcpkg4\buildtrees\atk\arm64-windows-rel\meson-logs\meson-log.txt
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "1",
  "version": "1.0",
  "dependencies": [
    {
      "name": "atk",
      "default-features": false,
      "features": [
        "introspection"
      ]
    }
  ]
}

```
</details>

--- a/ports/atk/portfile.cmake
+++ b/ports/atk/portfile.cmake
@@ -10,9 +10,6 @@ vcpkg_from_gitlab(
 )
 
 if("introspection" IN_LIST FEATURES)
-    if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-        message(FATAL_ERROR "Feature introspection currently only supports dynamic build.")
-    endif()
     list(APPEND OPTIONS_DEBUG -Dintrospection=false)
     list(APPEND OPTIONS_RELEASE -Dintrospection=true)
 else()

--- a/ports/atk/vcpkg.json
+++ b/ports/atk/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "atk",
   "version": "2.38.0",
-  "port-version": 7,
+  "port-version": 8,
   "description": "GNOME Accessibility Toolkit",
   "homepage": "https://developer.gnome.org/atk/",
   "license": "LGPL-2.0-or-later",
@@ -30,14 +30,7 @@
     "introspection": {
       "description": "build with introspection",
       "dependencies": [
-        {
-          "name": "gobject-introspection",
-          "host": true
-        },
-        {
-          "name": "gobject-introspection",
-          "platform": "windows & x86"
-        }
+        "gobject-introspection"
       ]
     }
   }

--- a/ports/gdk-pixbuf/portfile.cmake
+++ b/ports/gdk-pixbuf/portfile.cmake
@@ -13,9 +13,6 @@ vcpkg_from_gitlab(
 )
 
 if("introspection" IN_LIST FEATURES)
-    if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-        message(FATAL_ERROR "Feature introspection currently only supports dynamic build.")
-    endif()
     list(APPEND OPTIONS_DEBUG -Dintrospection=disabled)
     list(APPEND OPTIONS_RELEASE -Dintrospection=enabled)
 else()

--- a/ports/gdk-pixbuf/vcpkg.json
+++ b/ports/gdk-pixbuf/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gdk-pixbuf",
   "version": "2.42.10",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Image loading library.",
   "homepage": "https://gitlab.gnome.org/GNOME/gdk-pixbuf",
   "license": "LGPL-2.1-or-later",
@@ -28,14 +28,7 @@
     "introspection": {
       "description": "build with introspection",
       "dependencies": [
-        {
-          "name": "gobject-introspection",
-          "host": true
-        },
-        {
-          "name": "gobject-introspection",
-          "platform": "windows & x86"
-        }
+        "gobject-introspection"
       ]
     }
   }

--- a/ports/gtk/portfile.cmake
+++ b/ports/gtk/portfile.cmake
@@ -33,9 +33,6 @@ list(APPEND OPTIONS -Dwin32-backend=${win32}) #Enable the Windows gdk backend (o
 list(APPEND OPTIONS -Dmacos-backend=${osx}) #Enable the macOS gdk backend (only when building on macOS)
 
 if("introspection" IN_LIST FEATURES)
-    if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-        message(FATAL_ERROR "Feature introspection currently only supports dynamic build.")
-    endif()
     list(APPEND OPTIONS_DEBUG -Dintrospection=disabled)
     list(APPEND OPTIONS_RELEASE -Dintrospection=enabled)
 else()

--- a/ports/gtk/vcpkg.json
+++ b/ports/gtk/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "gtk",
   "version": "4.10.5",
+  "port-version": 1,
   "description": "Portable library for creating graphical user interfaces.",
   "homepage": "https://www.gtk.org/",
   "license": "LGPL-2.0-only",
@@ -72,14 +73,6 @@
           "features": [
             "introspection"
           ]
-        },
-        {
-          "name": "gobject-introspection",
-          "host": true
-        },
-        {
-          "name": "gobject-introspection",
-          "platform": "windows & x86"
         },
         {
           "name": "graphene",

--- a/ports/gtk3/vcpkg.json
+++ b/ports/gtk3/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "gtk3",
   "version": "3.24.38",
+  "port-version": 1,
   "description": "Portable library for creating graphical user interfaces.",
   "homepage": "https://www.gtk.org/",
   "license": null,
@@ -77,14 +78,6 @@
           "features": [
             "introspection"
           ]
-        },
-        {
-          "name": "gobject-introspection",
-          "host": true
-        },
-        {
-          "name": "gobject-introspection",
-          "platform": "windows & x86"
         },
         {
           "name": "pango",

--- a/ports/harfbuzz/portfile.cmake
+++ b/ports/harfbuzz/portfile.cmake
@@ -35,9 +35,6 @@ list(APPEND FEATURE_OPTIONS -Dfreetype=enabled) #Enable freetype interop helpers
 #endif()
 
 if("introspection" IN_LIST FEATURES)
-    if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-        message(FATAL_ERROR "Feature introspection currently only supports dynamic build.")
-    endif()
     list(APPEND OPTIONS_DEBUG -Dgobject=enabled -Dintrospection=disabled)
     list(APPEND OPTIONS_RELEASE -Dgobject=enabled -Dintrospection=enabled)
 else()

--- a/ports/harfbuzz/vcpkg.json
+++ b/ports/harfbuzz/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "harfbuzz",
   "version": "8.2.0",
+  "port-version": 1,
   "description": "HarfBuzz OpenType text shaping engine",
   "homepage": "https://github.com/harfbuzz/harfbuzz",
   "license": "MIT-Modern-Variant",
@@ -50,14 +51,7 @@
       "description": "build with introspection",
       "dependencies": [
         "glib",
-        {
-          "name": "gobject-introspection",
-          "host": true
-        },
-        {
-          "name": "gobject-introspection",
-          "platform": "windows & x86"
-        }
+        "gobject-introspection"
       ]
     }
   }

--- a/ports/pango/portfile.cmake
+++ b/ports/pango/portfile.cmake
@@ -15,9 +15,6 @@ vcpkg_from_gitlab(
 vcpkg_replace_string("${SOURCE_PATH}/meson.build" "-Werror=array-bounds" "")
 
 if("introspection" IN_LIST FEATURES)
-    if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-        message(FATAL_ERROR "Feature introspection currently only supports dynamic build.")
-    endif()
     list(APPEND OPTIONS_DEBUG -Dintrospection=disabled)
     list(APPEND OPTIONS_RELEASE -Dintrospection=enabled)
 else()

--- a/ports/pango/vcpkg.json
+++ b/ports/pango/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "pango",
   "version": "1.50.14",
-  "port-version": 3,
+  "port-version": 4,
   "description": "Text and font handling library.",
   "homepage": "https://ftp.gnome.org/pub/GNOME/sources/pango/",
   "license": "LGPL-2.0-or-later",
@@ -39,14 +39,6 @@
     "introspection": {
       "description": "build with introspection",
       "dependencies": [
-        {
-          "name": "gobject-introspection",
-          "host": true
-        },
-        {
-          "name": "gobject-introspection",
-          "platform": "windows & x86"
-        },
         {
           "name": "harfbuzz",
           "features": [

--- a/versions/a-/atk.json
+++ b/versions/a-/atk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c97f8dd9d133fb3e75abf33040b3ec61e032734d",
+      "version": "2.38.0",
+      "port-version": 8
+    },
+    {
       "git-tree": "d21ac2c541d67128cafda7c4b190e217cb943174",
       "version": "2.38.0",
       "port-version": 7

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -290,7 +290,7 @@
     },
     "atk": {
       "baseline": "2.38.0",
-      "port-version": 7
+      "port-version": 8
     },
     "atkmm": {
       "baseline": "2.36.1",
@@ -2810,7 +2810,7 @@
     },
     "gdk-pixbuf": {
       "baseline": "2.42.10",
-      "port-version": 1
+      "port-version": 2
     },
     "gemmlowp": {
       "baseline": "2021-09-28",
@@ -3070,11 +3070,11 @@
     },
     "gtk": {
       "baseline": "4.10.5",
-      "port-version": 0
+      "port-version": 1
     },
     "gtk3": {
       "baseline": "3.24.38",
-      "port-version": 0
+      "port-version": 1
     },
     "gtkmm": {
       "baseline": "4.10.0",
@@ -3154,7 +3154,7 @@
     },
     "harfbuzz": {
       "baseline": "8.2.0",
-      "port-version": 0
+      "port-version": 1
     },
     "hash-library": {
       "baseline": "8",
@@ -6258,7 +6258,7 @@
     },
     "pango": {
       "baseline": "1.50.14",
-      "port-version": 3
+      "port-version": 4
     },
     "pangolin": {
       "baseline": "0.8",

--- a/versions/g-/gdk-pixbuf.json
+++ b/versions/g-/gdk-pixbuf.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "3dc0bb4b0113c8043f2ed716694303d1bdc08d88",
+      "version": "2.42.10",
+      "port-version": 2
+    },
+    {
       "git-tree": "ca75f73ba67cdacfd245b67e6dca8c45fb97ee93",
       "version": "2.42.10",
       "port-version": 1

--- a/versions/g-/gtk.json
+++ b/versions/g-/gtk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "814624705f6cc11a3468d9c3418bc20521b6b207",
+      "version": "4.10.5",
+      "port-version": 1
+    },
+    {
       "git-tree": "1d2910bafafe5342d557621dfb784fab156ae169",
       "version": "4.10.5",
       "port-version": 0

--- a/versions/g-/gtk3.json
+++ b/versions/g-/gtk3.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1362bb3978886e043fd12c089f2c1337a4f1adf8",
+      "version": "3.24.38",
+      "port-version": 1
+    },
+    {
       "git-tree": "63636acf7b77eead112fc9faa84680d1df97acd3",
       "version": "3.24.38",
       "port-version": 0

--- a/versions/h-/harfbuzz.json
+++ b/versions/h-/harfbuzz.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "053b71e591f5599aa5e887fbf6531c7baa58b23d",
+      "version": "8.2.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "d804324af44593c0877fdf035cda178123d9c87f",
       "version": "8.2.0",
       "port-version": 0

--- a/versions/p-/pango.json
+++ b/versions/p-/pango.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "e2a88ce998fab5ed3f3a2edf817f32734c78187b",
+      "version": "1.50.14",
+      "port-version": 4
+    },
+    {
       "git-tree": "4c3bbb58011a3e259be06531c08854dfd7bbabee",
       "version": "1.50.14",
       "port-version": 3

